src/Entity/OgRole.php

```diff
@@ -187,6 +222,10 @@ public function save() {
         throw new ConfigValueException('The group bundle can not be empty.');
       }
 
+      if (empty($this->getName())) {
+        throw new ConfigValueException('The role name can not be empty.');
```

Comment:
we seem to have a similar check in `validate`, however that one throws a different exception. Maybe consolidate it, and call validate() from here instead?